docs/user_guide/assessment/common_fairness_metrics.rst

```diff
@@ -299,7 +303,122 @@ capture the costs of missclassification disparities.
+application may also give an unjustified feeling of legal invulnerability by
+conflating fairness with legality.
+In reality, 'fairness' is not always identical to 'legally allowable,' and
+the former may not even be a strict subset of the latter. [#f1]_
+
+The 'four fifths rule' has its origins in a specific area of US
+federal employment law.
+It is a limit for
+`prima facie evidence <https://en.wikipedia.org/wiki/Prima_facie>`_
+that illegal discrimination has occurred relative to a 
+relevant control population.
+A violation of the rule is still not sufficient to demonstrate that
+illegal discrimination has occurred - a causal link between the
+statistic and alleged discrimination must still be shown, and the
+(US) court would examine the particulars of each case.
+For an example of the subtleties involved, see
+`Ricci v. Stefano <https://en.wikipedia.org/wiki/Ricci_v._DeStefano>`_
+which resulted from an attempt to 'correct' for disparate impact.
+*Outside* its particular context in US federal employment law,
+the 'four fifths rule' has no validity.
+
+Taken together, we see that applying the 'four fifths rule' will
+not be appropriate in most cases.
+Even in cases where it is applicable, the rule does not automatically
+avoid legal jeopardy, much less ensure that results are fair.
+The use of the 'four fifths rule' in this manner is an indefensible
+example of epistemic trespassing. [#f2]_
+It is for this reason that we try to avoid the use of legal
+terminology in our documentation.
+
+For a much deeper discussion of the issues involved, we suggest
+:footcite:ct:`watkins2022fourfifths`.
+A higher level look at how legal concepts of fairness can collide
+with mathematical measures of disparity, see
+:footcite:ct:`Xiang2019legalcompatibility`.
+
+
+Summary
+-------
+
+We have introduced three commonly used fairness metrics in this section,
+which can be summed up as follows:
+
+* Demographic Parity
+
+    * *What it compares:* Predictions between different groups
+      (true values are ignored)
+
+    * *Reason to use:* If the input data are known to contain historical
+      biases, demographic parity may be appropriate
+
+    * *Caveats:* By only using the predicted values, information is thrown
+      away
+
+* Equalized Odds
+
+    * *What it compares:* True and False Positive rates between different groups
+
+    * *Reason to use:* If historical data are useful, and true and false positives
+      have roughly equal weight, equalized odds may be useful
+
+    * *Caveats:* If there are historical biases in the data, then the original labels
+      may hold little value. Also, large label imbalances (e.g. 95% of the data have 
+      a positive label) may make equalized odds an inappropriate measure
+
+* Equal opportunity
+
+    * *What it compares:* True Positive rates between different groups
+
+    * *Reason to use:* If historical data are useful, and false positive rates
```

Comment:
Nitpick: I think you'd want to use it not because FPR is *less* important but because true positives (and therefore TPR) correspond to something that is important for decision subjects, e.g., because it quantifies whether somebody received a benefit they are deserving of and we want to avoid false negatives (i.e., people "missing out" on a positive.

Reply:
Adjusted to talk about harms